### PR TITLE
Carousel clean

### PR DESCRIPTION
**Pull request by jassmith**
Remove CarouselView for preparation in moving to its own nuget.

### CODE REVIEW

**Comment by JoshuaNovak919** [comment]
@jassmith Any idea when the CarouselView nuget will be released? I need to use it in my project and i'm trying to see if I need to find a different way to do it, or if the nuget will be released soon.

**Comment by JoshuaNovak919** [comment]
@jassmith ???

**Comment by rmarinho** [comment]
@JoshuaNovak919 soon, hard yo give you a date but we are working hard on it, we had a issue that we are still trying to figure out.

File: Xamarin.Forms.Controls/GalleryPages/CarouselViewGallery.cs [hunk 1]
```diff
@@ -1,366 +0,0 @@
...
-
-#if UITEST
-using Xamarin.UITest;
-using NUnit.Framework;
-#endif
-
-namespace Xamarin.Forms.Controls
-{
-	[Preserve(AllMembers = true)]
-	public sealed class CarouselViewGallaryPage : ContentPage // or TestMasterDetailPage, etc ...
```
Comment (ghuntley) [inline Xamarin.Forms.Controls/GalleryPages/CarouselViewGallery.cs]: CarouselViewGall*a*ryPage  :thought_balloon:

**Pull request merged**